Graphics/Vulkan/VulkanGraphicsRenderPass.cpp

```diff
@@ -1,11 +1,16 @@
 // Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
 // for more information.
 
+#include <NovelRT/Graphics/GraphicsRenderPass.hpp>
+
 #include <NovelRT/Graphics/Vulkan/VulkanGraphicsRenderPass.hpp>
 
-namespace NovelRT::Graphics::Vulkan
+namespace NovelRT::Graphics
 {
-    VulkanGraphicsRenderPass::VulkanGraphicsRenderPass(VkRenderPass renderPass) noexcept : _vulkanRenderPass(renderPass)
+    using VulkanGraphicsRenderPass = GraphicsRenderPass<Vulkan::VulkanGraphicsBackend>;
+
+    VulkanGraphicsRenderPass::GraphicsRenderPass(VkRenderPass renderPass) noexcept
```

Comment:
What's up with the namespace here?

Reply:
GraphicsRenderPass is in NovelRT::Graphics. This is the implementation for the vulkan specialization of that type.